common/windows/delphi/packages/PackageInfo.pas

```diff
@@ -697,16 +708,23 @@ procedure TPackageOptions.LicenseRemoved(Sender: TObject; EventType: TPackageNot
   FLicenseFile := nil;
 end;
 
+procedure TPackageOptions.WelcomeRemoved(Sender: TObject; EventType: TPackageNotifyEventType; var FAllow: Boolean);
```

Comment:
How doe the FAllow boolean fit in with these procedures?

Reply:
The `FAllow` var parameter is an output parameter, allowing the function to prevent the removal of the item, which we never need to do, so it's safely ignored.